<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Железнодорожный вокзал</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/resourse/train.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="Table_flight">
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Refresh">
        <property name="text">
         <string>Обновить данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Sell_or_Buy">
        <property name="text">
         <string>Продать/купить билет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Return_Ticket">
        <property name="text">
         <string>Возврат билета</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="About"/>
    <addaction name="Logout"/>
    <addaction name="Exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="About">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/resourse/about.png</normaloff>:/resourse/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="Logout">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/resourse/exit_for_user.png</normaloff>:/resourse/exit_for_user.png</iconset>
   </property>
   <property name="text">
    <string>Выйти из системы</string>
   </property>
  </action>
  <action name="Exit">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/resourse/exit_from_program.png</normaloff>:/resourse/exit_from_program.png</iconset>
   </property>
   <property name="text">
    <string>Выйти из программы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
